lib/pluginmanager/install.rb

```diff
@@ -123,12 +123,27 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    gems_swap = {}
     options = { :rubygems_source => gemfile.gemset.sources }
     gems.each do |plugin, version|
       puts("Validating #{[plugin, version].compact.join("-")}")
       next if validate_plugin(plugin, version, options)
+      # if the plugin is alias fallback to the original name
+      if LogStash::PluginManager::ALIASES.has_key?(plugin)
+        aliased_plugin = LogStash::PluginManager::ALIASES[plugin]
+        if validate_plugin(aliased_plugin, version, options)
+          puts "Remapping alias #{plugin} to #{aliased_plugin}"
+          gems_swap[plugin] = aliased_plugin
```

Comment:
nit: variable naming

```suggestion
        resolved_plugin = LogStash::PluginManager::ALIASES[plugin]
        if validate_plugin(resolved_plugin, version, options)
          puts "Remapping alias #{plugin} to #{resolved_plugin}"
          gems_swap[plugin] = resolved_plugin
```